AndroidAnnotations/androidannotations-ormlite/ormlite-api/src/main/java/org/androidannotations/ormlite/api/OrmLiteInjectHelper.java

```diff
@@ -0,0 +1,8 @@
+package org.androidannotations.ormlite.api;
+
+/**
+ * @param <T> type to inject
+ */
+public interface OrmLiteInjectHelper<T> {
+	T inject();
+}
```

Comment:
What is the problem? This was the same with field injection. We wrapped the inejction in try-catch blocks. Now we have to wrap the setter method call in a try-catch block. Or is that hard to implement?